src/components/User/User.tsx

```diff
@@ -0,0 +1,121 @@
+import React from 'react';
+import { Input, Row, Button, Checkbox, Typography, Col, Form } from 'antd';
+
+import './styles.css';
+  
+interface MyFormValues {
+    firstName: string;
+    lastName: string;
+    email: string;
+    password: string;
+    newsletter: boolean;
+}
+
+export function User() {
```

Comment:
Form values will come from props, so U need to define an interface for component and set default props. Than use like initialValues.

Also I recommend to create method getInitialValues from props because props could be different from initialValues from props, also I recommend to create const inside component like fromRules to keep all validation logic in one place